spec/editors_draft.html

```diff
@@ -3298,6 +3298,261 @@ <h2>Evaluation Semantics</h2>
+        </li>
+        <li>
+          if the format of the document being processed does not have a mechanism for explicitly identifying blank nodes outside of cdt:List and cdt:Map literals, then <var>bnl'</var> must be picked such that <i><a>bnl2bn</a></i>(<var>bnl'</var>) is a fresh blank node not already used in the SPARQL system or RDF Dataset, and
+        </li>
+        <li>within the scope of the document being processed, all occurrences of <var>bnl</var> used to represent a blank node identifier within the lexical forms of cdt:List and cdt:Map literals are replaced with the same <var>bnl'</var> value, and</li>
+        <li>within the scope of the document being processed, no other substring that matches the <a data-cite="TURTLE#grammar-production-BLANK_NODE_LABEL">`BLANK_NODE_LABEL`</a> production within the lexical form of a cdt:List or cdt:Map literal is replaced by the same substring <var>bnl'</var>.</li>
+        </li>
+      </ul>
+
+      <p>
+        Here, "loading" is every individual, discrete process by which composite values enter a SPARQL system or RDF Dataset. This includes (but is not limited to):
+      </p>
+
+      <ul>
+        <li>The parsing of a SPARQL Query string [[SPARQL11-QUERY]] or of a SPARQL Update string [[SPARQL11-UPDATE]], which may contain literals.</li>
+        <li>The loading of RDF data via a SPARQL <a data-cite="SPARQL11-UPDATE#load">`LOAD`</a> operation [[SPARQL11-UPDATE]]</li>
+        <li>The loading of RDF data via a single SPARQL graph store protocol request [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of RDF data via any single SPARQL <a data-cite="SPARQL11-QUERY#unnamedGraph">`FROM`</a> or <a data-cite="SPARQL11-QUERY#namedGraphs">`FROM NAMED`</a> clause [[SPARQL11-QUERY]] in systems that <a data-cite="SPARQL11-SERVICE-DESCRIPTION#sd-dereferencesuris">support the dereferencing and loading of RDF data specified for the dataset to be queried</a></li>
+        <li>Importing of any RDF data to the internally-managed RDF dataset via means other than SPARQL Update statements [[SPARQL11-UPDATE]] or the SPARQL Graph Store HTTP Protocol [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of a SPARQL result obtained from a SPARQL service when evaluating a <a data-cite="SPARQL11-FEDERATED-QUERY#service">`SERVICE`</a> pattern [[SPARQL11-FEDERATED-QUERY]]</li>
+      </ul>
+
+      <p>
+        As an example, consider the Turtle document in <a href="#load-renaming-blank-example-simple"></a>.
+        Loading this document MUST result in a system-internal representation of a single triple whose object is a <a>cdt:List literal</a> containing a blank node that, in the lexical form of the literal, is captured by a blank node identifier that is <em>not</em> "b0" (e.g., <a href="#load-renaming-blank-example-simple-result"></a>).
```

Comment:
I don't see how the last part of this sentence (i.e., the bnode identifier must not be "b0"), is a consequence of  the import requirements that we currently have.

Reply:
OK. Here's my proposed rewriting:

> As an example, consider the Turtle document in Example 44. Loading this document MUST result in a system-internal representation of a single triple whose object is a cdt:List literal containing a blank node that, in the lexical form of the literal, is captured by a blank node identifier which may have been replaced during import (e.g., Example 45).

It is very weak in its prescription, but I think it's still important as a first example showing that the lexical form may change. Thoughts?

Reply:
Yes, that makes sense. Please commit this version.

Reply:
Here's your version as an edit suggestion. I will commit it.
```suggestion
        Loading this document MUST result in a system-internal representation of a single triple whose object is a <a>cdt:List literal</a> containing a blank node that, in the lexical form of the literal, is captured by a blank node identifier that may have been replaced during import (e.g., <a href="#load-renaming-blank-example-simple-result"></a>).
```